CHANGELOG.md

```diff
@@ -127,6 +127,7 @@ We are happy to announce the following GA libraries.  Unless specifically noted,
 the APIs in these libraries are stable, and are ready for production use.
 
 - [Cloud Support API](/google/cloud/support/README.md)
+- [Discovery Engine API](/google/cloud/discoveryengine/README.md)
```

Comment:
The service is not GA yet (soon I expect, but not yet):

https://cloud.google.com/discovery-engine/media/docs/introduction

![image](https://github.com/googleapis/google-cloud-cpp/assets/6241635/c604f040-7283-49f2-885c-df9e8ae299ed)